oio/directory/meta.py

```diff
@@ -118,6 +126,11 @@ def _apply_copy_bases(self, moved, **kwargs):
                         all_peers_ok = False
                 if all_peers_ok:
                     moved_ok.append(base)
+                    if service_type == 'meta2' and kwargs.get('src_service'):
+                        self.rdir.meta2_index_delete(
+                            volume_id=kwargs.get('src_service'),
+                            container_id=cid
+                        )
```

Comment:
At this point, the old peer it still referenced in the directory. If a crawler walks on it, it will be referenced again. This workaround should be done after `_apply_link_services`.

Also, I would like it to be clear: this is a workaround and definitely not the best solution to deindex the database.